### PR TITLE
Move admin nav to navbar

### PR DESCRIPTION
**Pull request by Sigevio**

### CODE REVIEW

**Comment by pavelskipenes** [comment]
Is this intended? `admin.nav.store`
image
Otherwise LGTM

File: components/navbar/content/AdminNav.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    {
+      label: t('admin.nav.users'),
+      value: 'admin/users'
+    },
+    {
+      label: t('admin.nav.members'),
+      value: 'admin/members'
+    },
+    {
+      label: t('admin.nav.store'),
```
Comment (pavelskipenes) [inline components/navbar/content/AdminNav.tsx:29]: I guess the translations are just missing...

File: i18n/translations/en.json [hunk 2]
```diff
@@ -7,6 +7,12 @@
       "password": "Password",
       "loginFailed": "Login failed!",
       "button": "Send"
+    },
+    "nav": {
+      "profile": "My profile",
+      "users": "Users",
+      "members": "Members",
+      "webStore": "Store"
```
Comment (pavelskipenes) [inline i18n/translations/en.json:15]:
```suggestion
      "store": "Store"
```

Comment (Sigevio) [reply i18n/translations/en.json:15]: Woopsie! Forgot to update that key in the english translations:)

**Pull request merged**